waterlog/src/Components/Wastage.js

```diff
@@ -55,11 +55,14 @@ class Wastage extends Component {
       seasonalLoading,
       placeholderLoading,
       placeholderError,
+      dailyWaste,
+      monthlyWaste,
+      seasonWaste,
```

Comment:
Please note the comma after seasonWaste as it is the last item

Reply:
removed , please re-check